movieapp/src/containers/AddEditMovieForm/AddEditMovieForm.tsx

```diff
@@ -0,0 +1,79 @@
+import {Button} from "../../components/Button/Button";
+import {useTranslation} from "react-i18next";
+
+type MovieProps = {
+    title: string | '';
+    description: string | '';
+    key: string | '';
```

Comment:
do not use `key` in model. use something as `id` or `pk` (primaryKey)

cause key is like reseved naming in react and it's hard to understand whether it's react key or model property

Reply:
fixed